### DISCUSSION TITLE
Remove humidity and temperature indicator

### DISCUSSION THREAD

**Discussion by mutchu** [merdok/homebridge-miot#325] [General]
Hello and thank you for your work,
Is there a way for air purifiers to remove the humidity and temperature indicator so they don't show up in homekit?
Thank you in advance for your return.
Cordially

**Comment by merdok** [comment]
Hi,
it is not possible currently. As long as a device supports that features, does not matter what kind of device, they will always appear. 
May i ask why you want to disable them?

**Comment by mutchu** [comment]
Hello, thank you for your feedback. I want to disable it because it conflicts with my Netatmo weather station certified for HomeKit. And I would like to keep only the data of the Netatmo. Thanks

Cordialement 

Michel BERLIE

**Reply by merdok** [reply]
You can try to use the onlyMainService configuration property and set it to true. That might help you out.

**Reply by mutchu** [reply]
How to do this ?

**Reply by merdok** [reply]
Please read the README

**Reply by mutchu** [reply]
Where ?

**Reply by merdok** [reply]
Just have a look at the README, all the options are described there. You will also find that option there.
